modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1280,6 +1280,12 @@ AUI.add(
 					if (value) {
 						var date = A.DataType.Date.parse(value);
 
+						if (!date) {
+							datePicker.selectDates('');
```

Comment:
Hey @NemethNorbert,

Set an empty date when some browsers can't do the parse won't generate any side effect?

Reply:
Hey @natocesarrego,

It should not. If the 'value' is null, we do the same here: https://github.com/natocesarrego/liferay-portal/blob/1b080120355c9786d6b06c58b40d08e85ee4025f/modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js#L1297

Reply:
I see, @NemethNorbert. But in the code shared by you there is no value to be added, on the other hand on the issue's use case there is a value, despite it can't be parsed by FF and IE. Have you had the chance to check if the current fix doesn't introduce any issue on the Web Content view/rendering?

Sorry for make this review take longer, but I want to make sure we're not introducing any regressions.

Thanks for your patience.

Reply:
Hey @natocesarrego,
Sure I tested the fix again, and I did not see any regression, during saving, editing and rendering in FF, Chrome and Safari (I could not test in IE).
I think it safe to merge.
Thanks

Reply:
Thanks, @NemethNorbert! I'll test on IE and if everything goes fine, then I forward you pull request. 🙂